src/lib/is_tap.js

```diff
@@ -0,0 +1,16 @@
+const euclideanDistance = require('./euclidean_distance');
+
+const TOLERANCE = 25;
+const INTERVAL = 250;
+
+module.exports = function isTap(start, end, options = {}) {
```

Comment:
> the real reason I split them is because I felt that touch events do not need to gross/fine tolerance mechanism because maybe a finger is less likely to move once it has it the screen

This makes sense to me.

> calling is_click inside a touch handler might seem confusing to the reader but perhaps renaming it to something that an is_click and is_tap could both call would work

What about [checkInteraction](https://gist.github.com/mcwhittemore/74143b310bb70e58f326f82500707b90) which exports a `isClick` and a `isTap` function.